### PR TITLE
Image migration to static resources

### PR DESCRIPTION
**Pull request by AlekhyaSasi**
Signed-off-by: Alekhya Sai Punnamaraju <alekhyasaip@gmail.com>

# TL;DR
- Migrating the Images to Static resources.
- Grammar fix to Readme in Proposals.

## Type
- [ ] Bug Fix
- [x] Feature
- [ ] Plugin

## Complete description
Related PR - https://github.com/flyteorg/static-resources/pull/3
Related Issue - https://github.com/flyteorg/flyte/issues/1986

### CODE REVIEW

File: proposal/README.md [hunk 1]
```diff
@@ -144,23 +145,24 @@ $ flytectl update projects --id project-x ...
 ```
 
 ## Tasks
- - get
+
```
Comment (samhita-alla) [inline proposal/README.md:148]: Revert?

File: proposal/README.md [hunk 2]
```diff
@@ -68,25 +70,23 @@ $ flytectl --endpoint "example.flyte.net" --project "p" --domain "d" delete work
 ```
 
 ### Create is special
-Create may need more information than can be easily passed in command line and we recommend using files to create an entity. The file could be in protobuf, jsonpb (json) or jsonpb (yaml) form.
-Eventually we may want to simplify the json and yaml representations but that is not required in first pass. We may also want to create just a separate option for that.
-
-The create for Task and Workflow is essential what is encompassed in the pyflyte as the registration process. We will decouple the registration process such that pyflyte, jflyte (other native cli's or
-code methods) can dump a serialized representations of the workflows and tasks that are directly consumed by **Flytectl**. Thus Flytectl is essential in every flow for the user.
 
+The create for Task and Workflow is essential to what is encompassed in the pyflyte as the registration process. We will decouple the registration process such that the pyflyte, jflyte (other native cli's or
```
Comment (samhita-alla) [inline proposal/README.md:74]: Have you removed the previous paragraph intentionally?

Comment (AlekhyaSasi) [reply proposal/README.md:74]: Isn't it a repetitive paragraph?

File: proposal/README.md (hunk 1 above)
Comment (AlekhyaSasi) [reply proposal/README.md:148]: I didn't get you. Do you want me to revert this change?

Comment (samhita-alla) [reply proposal/README.md:148]: Yes!

File: proposal/README.md (hunk 2 above)
Comment (samhita-alla) [reply proposal/README.md:74]: I don't think so.

Comment (AlekhyaSasi) [reply proposal/README.md:74]: Got it. Let me revert. Thank you.

**Review by samhita-alla — APPROVED** [review]

**Pull request merged**